### PR TITLE
Add content in an SPA model

### PR DESCRIPTION
**Pull request by maaloum**
### In this PR, I :
-  Created a SPA code using JavaScript and CSS.
- The code is toggling between different sections.
- when you click on any of the links, the main area changes to display only one of the three content sections.

### CODE REVIEW

File: js/spa.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  e.preventDefault();
+  const t = document.querySelector('#title').value;
+  const a = document.querySelector('#author').value;
+  // const book = new Book(t, a);
+  if (!(t === '' && a === '')) {
+    const newBook = {
+      title: t,
+      author: a,
+    };
+    addBook(newBook);
```
Comment (muneebulrehman) [inline js/spa.js:86]: When I add multiple books and then delete a book, followed by reloading the page and then adding another book, the older data is overridden by the recently added book. Consider fixing it.

File: index.html [hunk 2]
```diff
@@ -4,24 +4,72 @@
...
+    </p>
+      <ul>
+        <li>Our email: Awesomebooks@microverse.com</li>
+        <li>Our phone : 00441789516</li>
+        <li>Our adress : Casabanca microverse</li>
+
+      </ul>
+  </section>
+
+  <footer>
```
Comment (muneebulrehman) [inline index.html:53]: The footer does not follow the wireframe design provided. 

| Your design  | Provided design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @maaloum , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @maaloum , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**